Mathlib/NumberTheory/LegendreSymbol/AddCharacter.lean

```diff
@@ -296,7 +296,9 @@ private lemma ringChar_ne : ringChar ℂ ≠ ringChar F := by
 /-- A primitive additive character on the finite field `F` with values in `ℂ`. -/
 noncomputable def FiniteField.primitiveChar_to_Complex : AddChar F ℂ := by
   refine MonoidHom.compAddChar ?_ (primitiveChar F ℂ <| ringChar_ne F).char
-  exact (IsCyclotomicExtension.algEquiv ?n ℂ (CyclotomicField ?n ℂ) ℂ : CyclotomicField ?n ℂ →* ℂ)
+  exact (IsCyclotomicExtension.algEquiv ↑(primitiveChar F ℂ (ringChar_ne F)).n
+    ℂ (CyclotomicField (↑(primitiveChar F ℂ (ringChar_ne F)).n) ℂ) ℂ :
+    CyclotomicField ↑(primitiveChar F ℂ _).n ℂ →* ℂ)
```

Comment:
Maybe do
```lean
  letI ch := primitiveChar F ℂ <| ringChar_ne F
  refine  MonoidHom.compAddChar ?_ ch.char
  exact (IsCyclotomicExtension.algEquiv ch.n ℂ (CyclotomicField ch.n ℂ) ℂ : CyclotomicField ch.n ℂ →* ℂ)
```
(Here the `letI` makes sense; it inlines the expression, which for data usually is better than to have a `let` hanging around.)